tag_argument_extractor.go

```diff
@@ -0,0 +1,188 @@
+package faker
+
+import (
+	"fmt"
+	"math/rand"
+	"strconv"
+	"strings"
+)
+
+func extractFloat64FromTagArgs(args []string) (interface{}, error) {
+	bytes := 64
+	var floatValues []float64
+	for _, i := range args {
+		k := strings.TrimSpace(i)
+		j, err := strconv.ParseFloat(k, bytes)
+		if err != nil {
+			return nil, fmt.Errorf(ErrUnsupportedTagArguments)
+		}
+		floatValues = append(floatValues, j)
+	}
+	toRet := floatValues[rand.Intn(len(floatValues))]
+	return toRet, nil
+}
+
+func extractFloat32FromTagArgs(args []string) (interface{}, error) {
+	bytes := 32
+	var floatValues []float32
+	for _, i := range args {
+		k := strings.TrimSpace(i)
+		j, err := strconv.ParseFloat(k, bytes)
+		if err != nil {
+			return nil, fmt.Errorf(ErrUnsupportedTagArguments)
+		}
+		floatValues = append(floatValues, float32(j))
+	}
+	toRet := floatValues[rand.Intn(len(floatValues))]
+	return toRet, nil
+}
+
+func extractInt64FromTagArgs(args []string) (interface{}, error) {
+	bytes := 64
+	var floatValues []int64
+	for _, i := range args {
+		k := strings.TrimSpace(i)
+		j, err := strconv.ParseInt(k, 0, bytes)
```

Comment:
any particular reasons to put the base to zero (0) instead of directly `10`? Is it because you want to make this support hex decimal as well?

Reply:
I'm sorry for the late reply, so busy lately 🙏

Reply:
sure

Reply:
actually I'm running out of steam and am getting more busy over here as well. I'm going to switch the base back to base 10 and leave adding hex/binary/octal/etc other base number types for another PR.